app/scripts/migrations/060.js

```diff
@@ -0,0 +1,30 @@
+import { cloneDeep } from 'lodash';
+
+const version = 60;
+const SUPPORT_NOTIFICATION_KEY = 2;
+const SUPPORT_NOTIFICATION_DATE = '2020-08-31';
+
+/**
+ * Removes the support survey notification
+ */
+export default {
+  version,
+  async migrate(originalVersionedData) {
+    const versionedData = cloneDeep(originalVersionedData);
+    versionedData.meta.version = version;
+    const state = versionedData.data;
+    const newState = transformState(state);
+    versionedData.data = newState;
+    return versionedData;
+  },
+};
+
+function transformState(state) {
+  if (
+    state?.NotificationController?.notifications[SUPPORT_NOTIFICATION_KEY]
```

Comment:
Nit: it would be great to have tests for all branches, such as the case where `NotificationController` is nullish.

Probably best to add a few more `?.` operators here as well, in case `notifications` is nullish, and in case `notifications[SUPPORT_NOTIFICATION_KEY]` is nullish. So that this won't crash if the state is invalid.

Reply:
`transformState` updated to accommodate instances where `NotificationController` or `notifications` are non-existent. Test cases added to cover additional branches